community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/slurmsync.py

```diff
@@ -68,6 +68,18 @@
 )
 
 
+KeepStates = frozenset(
```

Comment:
I wouldn't move it to global scope as it doesn't get used outside of the `sync_placement_groups`